cli/azd/cmd/models/show.go

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package models
```

Comment:
Seemed like it would be worthwhile to start extracting the model types that we return for out `--output=json` commands into a special package so it was clearer these are external contracts, and we can use a more critical eye when reviewing.

I plan to adopt this pattern elsewhere as we define these contracts for commands like `env refresh` and `provision`.  Interested in the team's thoughts on this pattern.

Reply:
I think that's a really good idea.

If we do go down this approach, does it make sense to add a package level doc? Something like:
```suggestion
// Package models contains API contracts that azd CLI communicates externally in commands via stdout. Currently, all contracts support JSON output.
package models
```

As I wrote the doc, I wonder if `contracts` is a better name over `models`

Reply:
I'm fine `contracts`.  I'll add a doc comment as well.